packages/plugins/auto-doc/src/utils/pdfGenerator.ts

```diff
@@ -202,27 +202,66 @@ function generatePdf(templateTitle: string , allBlocks: Element[]){
         const formattedHeader: string[][] = [data.map((row: string[]) => row[0])];
         const formattedBody: string[][] = [data.map((row: string[]) => row[1])];
 
-        const rows = data[0].length;
-        const tableHeight = rows * DEFAULT_LINE_HEIGHT + DEFAULT_LINE_HEIGHT;
+        const filledBody = formattedBody.map((row) => row.map(it => it = placeholderStore.fillPlaceholder(it)));
```

Comment:
does `it`stands for item ? Maybe a more describing name like `cell`could be better ?
```suggestion
        const filledBody = formattedBody.map((row) => row.map(cell => cell = placeholderStore.fillPlaceholder(cell)));
```